bin/des-pizza-cutter-fit-gaia-mask

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+
+import numpy as np
+import fitsio
+import hickory
```

Comment:
we should get hickory on conda-forge if you are agreeable to that. happy to do it myself. will need a license.

Reply:
Thanks.  But I don't want that, at least right now.